packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -159,8 +159,7 @@ abstract class FlutterCommand extends Command<void> {
   @override
   ArgParser get argParser => _argParser;
   final ArgParser _argParser = ArgParser(
-    allowTrailingOptions: false,
-    usageLineLength: globals.outputPreferences.wrapText ? globals.outputPreferences.wrapColumn : null,
+    usageLineLength: globals.outputPreferences.wrapText? globals.outputPreferences.wrapColumn : null,
```

Comment:
Was the removal of this space accidental?

Reply:
(and Thanks! 😄)

Reply:
oops!